js/interactive-guides/microprofile-config/playground-callback.js

```diff
@@ -0,0 +1,142 @@
+var playground = (function() {
+    
+    var properties = {};
+
+    /**
+     * 
+     * @param {*} key - 
+     * @param {*} value - 
+     * @param {*} source - 
+     * @param {*} ordinal (optional?) - provided ordinal number. otherwise default based on source
+     */
+    var playgroundAddConfig = function(key, value, source, ordinal) {
+        if (!ordinal) {
+            ordinal = __getDefaultOrdinal(source);
+        }
+
+        if (properties[key]) {
+            var oldOrdinal = properties[key]['ordinal'];
+            if (parseInt(ordinal) >= parseInt(oldOrdinal)) {
+                properties[key] = {'ordinal': ordinal, 'value': value};
+            }
+        } else {
+            properties[key] = {'ordinal': ordinal, 'value': value};
+        }
+    };
+
+    var repopulatePlaygroundConfigs = function(tabbedEditor) {
+        properties = {};
+
+        getInjectionProperties();
+        getPropertiesFileProperties();
+        getEnvironmentProperties();
+        getSystemProperties();
+    };
+
+    var getInjectionProperties = function() {
+        var injectionContent = contentManager.getTabbedEditorContents('DefaultPlayground', 'Injection');
+
+        var regexp = /@ConfigProperty/g;
+        var match, matches = [];
+        while ((match = regexp.exec(injectionContent)) != null) {
+            matches.push(match.index);
+        }
+        
+        var lines = [];
+        for (i in matches) {
+            var content = injectionContent.substring(matches[i]);
+            var endLine = content.indexOf(";");
+            var line = content.substring(0, endLine);
+            lines.push(line);
+        }
+
+        for (i in lines) {
```

Comment:
This will assign i to be a global variable which isn't used outside of this function. I think we should use a let or var.

Reply:
Fixed in 3a5df38